.travis.yml

```diff
@@ -19,7 +19,10 @@ before_install:
   - sudo make base
   - sudo make apt-openfoam-deps apt-simphony-deps apt-lammps-deps apt-mayavi-deps apt-aviz-deps
   - make fix-pip
-  - pip install --allow-external PIL --allow-unverified PIL PIL
+  - sudo pip uninstall -y argparse
```

Comment:
Yes I have tried ``pip install --upgrade argparse`` inside the simphony-common target.  In that case it would fail as the ``pip`` (in the virtual environment) complains about not knowing how to uninstalled an ``argparse`` package installed by ``distutil``.  I believe this is precisely why ``make simphony`` fails in the first place as it tries to update ``argparse``

I am not sure if ``sudo pip`` reinstall ``argparse`` is necessary generally.  I suspect this is a problem specific to packages come with Ubuntu 12.04 LTS.  Need to investigate more to find out where the dist-package's ``argparse`` come from.